src/server/json_family.cc

```diff
@@ -172,15 +169,16 @@ OpStatus UpdateEntry(const OpArgs& op_args, std::string_view key, std::string_vi
   // Make sure that we don't have other internal issue with the operation
   OpStatus res = verify_op(json_entry);
   if (res == OpStatus::OK) {
-    db_slice.PostUpdate(db_index, entry_it, key);
+    it_res->post_updater.Run();
```

Comment:
Honestly I wasn't even sure if we should explicitly `Run()` it here, as it would have run anyway right after `AddDoc()` below, but I didn't know if it's ok to run afterwards (although it probably is?)
@dranikpg can we run `PostUpdate()` after `AddDoc()`?

Reply:
> I'm not sure I follow your question. By calling post_updater.Run() you're running PostUpdate() now instead of in d'tor (it will not run in the d'tor after this)

Adi means that you don't prevent it from running even if opstatus wasn't OK, which didn't happen before

> @dranikpg can we run PostUpdate() after AddDoc()?

I assume so, because it only reads the value